completions/_dust

```diff
@@ -71,6 +71,7 @@ _dust() {
 '-V[Print version]' \
 '--version[Print version]' \
 '*::params:' \
+'*:inputs:_files' \
```

Comment:
This will not work. Remove the new line and replace the existing line with
```
'*::params:_files' \
```

Reply:
Happy to make this change. 

When I test tab completion it seems to work for me. 

How would you recommend that I test this? & How do you know this is the right thing to do ? 

Thanks

Reply:
Just edit the existing file in your fpath and then start a new zsh session. When I try with both `*:...` lines added:
```
❯ dust <tab>
_arguments:comparguments:327: doubled rest argument definition: *:inputs:_files
_arguments:comparguments:327: doubled rest argument definition: *:inputs:_files
❯ dust 
```
compared with the one line as in my suggestion:
```
❯ dust <tab>
-- params --
Desktop/  Documents/ ...
```